src/Core/Common/GraphQLRequest.cs

```diff
@@ -5,9 +5,11 @@ public class GraphQLRequest
         public const string OPERATION_NAME_KEY = "operationName";
         public const string QUERY_KEY = "query";
         public const string VARIABLES_KEY = "variables";
+        public const string EXTENSIONS_KEY = "extensions";
 
         public string OperationName { get; set; }
         public string Query { get; set; }
         public Inputs Inputs { get; set; }
+        public Inputs Extensions { get; set; }
```

Comment:
Using `Inputs` type here is a temporary measure. We can introduce a new type, although this one should fit.

Reply:
I think to keep `Inputs` type for this property.